product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/testng/listeners/TestPrepExecutionListener.java

```diff
@@ -174,7 +176,7 @@ private void undeployCarbonApplication(String applicationName)
     }
 
     private void setKeyStoreProperties() {
-        System.setProperty("javax.net.ssl.trustStore", resourceLocation + "/keystores/wso2carbon.jks");
+        System.setProperty("javax.net.ssl.trustStore", commonResourcesDir + "/keystores/wso2carbon.jks");
```

Comment:
If we move the commonsResourceDir to scenarios-commons, this might be a lot simpler

Reply:
Then tests will not be individually buildable. Need discussions with @milindaperera

Reply:
Moved to top level pom.xml as per discussions.